extension/js/common/core/buf.ts

```diff
@@ -97,6 +97,7 @@ export class Buf extends Uint8Array {
     return Buf.fromBase64Str(b64UrlStr.replace(/-/g, '+').replace(/_/g, '/'));
   };
 
+  // todo: deprecate
```

Comment:
I don't like this function and suggest to remove it. As in some places we might want to simplify code and use plain Uint8Array objects, and `Uint8Array.toString()` returns a totally different thing. (It's also hard to find usages of this function among `toString()` calls on other types of objects)
I suggest to explicitly use `toUtfStr` method)

Reply:
This one sounds good too 👍